.github/trigger_utils_tests

```diff
@@ -0,0 +1,17 @@
+name: Trigger AstroDB Utils Tests
+
+on:
+  push:
+    branches:
+      - main  # or whichever branches you want to monitor
+
+jobs:
+  trigger_repo1:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger AstroDB Utils Tests workflow
+        uses: peter-evans/repository-dispatch@v2
+        with:
+          token: ${{ secrets.UTILS_TOKEN }}
```

Comment:
this is a token in this repo (template)....maybe the token needs to be in astrodb_utils?